ros2/source/source/Dockerfile

```diff
@@ -72,17 +27,18 @@ RUN apt-get update && rosdep install -y \
     && rm -rf /var/lib/apt/lists/*
 
 # build source
-WORKDIR $ROS2_WS
 RUN colcon \
```

Comment:
Oh wait, are we missing a rosdep install here before building?

Reply:
The previous `RUN` statement (L23) is a `rosdep install`. Is that what you are referring to ?

Reply:
nvm, github collapsed those lines in the diff preview when I was skimming the latest changes.